conformance/base/manifests.yaml

```diff
@@ -91,9 +91,15 @@ spec:
   selector:
     app: infra-backend-v1
   ports:
-  - protocol: TCP
+  - name: first-port
+    protocol: TCP
     port: 8080
     targetPort: 3000
+  - name: third-port
```

Comment:
second port is going to be h2c

https://github.com/kubernetes-sigs/gateway-api/pull/2456/files#diff-21fe4de356049035b12df8b9efd2e0d29372189a548cd144caa363ca875e4f1dR98